packages/medusa/src/models/payment.ts

```diff
@@ -72,10 +72,10 @@ export class Payment {
   data: any
 
   @Column({ type: resolveDbType("timestamptz"), nullable: true })
-  captured_at: Date
+  captured_at: Date | string
```

Comment:
question: why is this necessary?

Reply:
Because a date can also be a string. For example toISOString will be a string 😅 generally speaking a date is always a date is a string and then when you use it you always pass it to the date constructor for later manipulation